js/scripts/generate-emoji-map.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+const path = require('path');
+const twemoji = require('twemoji/2/twemoji.npm');
+
+const data = require('emojibase-data/en/data.json');
+const twemojiFileNames = fs.readdirSync('./node_modules/twemoji/2/svg')
+  .map(name => path.basename(name, '.svg'));
+
+const alternative = {
```

Comment:
please see https://github.com/twitter/twemoji/issues/253
emojibase follows http://unicode.org/Public/emoji/ so it's not emojibase problem, but it seems `👁‍🗨` represented differently in various platforms, for instance macOS's emoji picker insert it without `FE0F`, so I'm also not sure if it's twemoji problem either. 

we can use this hack or we should ignore `👁‍🗨`.

Reply:
That hack is fine :)